README.md

```diff
@@ -36,7 +36,9 @@ You can reach the maintainers of this project at:
 
 - [Slack channel](https://kubernetes.slack.com/messages/cluster-addons)
```

Comment:
imo I would just point to the community repo as the canonical source of truth.

Reply:
Hrm. Where? I don't find any of these useful for anyone: https://github.com/kubernetes/community/search?q=cluster+addons+slack&unscoped_q=cluster+addons+slack

Reply:
You should be able to add all the deets https://github.com/kubernetes/community/tree/master/sig-cluster-lifecycle and just link